packages/core/src/modules/connections/__tests__/ConnectionService.test.ts

```diff
@@ -431,8 +431,8 @@ describe('ConnectionService', () => {
     it('returns a connection record containing the information from the connection response', async () => {
       expect.assertions(3)
 
-      const [did, verkey] = await wallet.createDid()
-      const [theirDid, theirVerkey] = await wallet.createDid()
+      const { did, verkey } = await wallet.createDid()
+      const { did: theirDid, verkey: theirVerkey } = await wallet.createDid()
```

Comment:
Is this intentional or symbol renaming?

Reply:
This is intentional. Using an object made more sense to me